src/OrbitBase/include/OrbitBase/SpanUtils.h

```diff
@@ -0,0 +1,46 @@
+// Copyright (c) 2022 The Orbit Authors. All rights reserved.
+// Use of this source code is governed by a BSD-style license that can be
+// found in the LICENSE file.
+
+#ifndef ORBIT_BASE_SPAN_UTILS_H_
+#define ORBIT_BASE_SPAN_UTILS_H_
+
+#include <absl/types/span.h>
```

Comment:
This should include <vector> and <cstddef> for size_t as well

Reply:
Added stddef.h